### PR TITLE
French translation of the `noopener` reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR creates the French translation of the `noopener` reference.

### CODE REVIEW

File: files/fr/web/html/link_types/noopener/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: 'Types de liens : noopener'
+slug: Web/HTML/Link_types/noopener
+browser-compat: html.elements.a.rel.noopener
+translation_of: 'Web/HTML/Link_types/noopener'
+---
+<p>Le mot-clé <strong><code>noopener</code></strong> de l'attribut <code><a href="/fr/docs/Web/HTML/Attributes/rel">rel</a></code> des éléments {{HTMLElement("a")}}, {{HTMLElement("area")}} et {{HTMLElement("form")}} indique au navigateur d'ouvrir la ressource cible sans donner d'accès au document précédent au nouveau contexte de navigation qui sera créé, en n'indiquant pas la propriété {{DOMxRef("Window.opener")}} sur la nouvelle fenêtre (il retourne <code>null</code>).</p>
+
+<p>Ceci est tout spécialement utile lors de l'ouverture de liens non vérifiés/fiables, afin de s'assurer qu'ils ne peuvent pas trafiquer le document d'origine via la propriété {{DOMxRef("Window.opener")}} (pour plus de détails, voir <a href="https://mathiasbynens.github.io/rel-noopener/">cet article en anglais concernant <code>rel=noopener</code></a>), lorsque l'en-tête HTTP <code>Referer</code> est fourni (à moins que <code>noreferrer</code> soit également utilisé).</p>
```
Comment (SphinxKnight) [inline files/fr/web/html/link_types/noopener/index.html:9]: I'd translate "while still providing the Referer HTTP header" with "tout en fournissant néanmoins l'en-tête HTTP Referer"

File: files/fr/web/html/link_types/noopener/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<p>Veuillez noter que lorsque <code>noopener</code> est utilisé, les noms de cibles non vides et différents de <code>_top</code>, <code>_self</code> et <code>_parent</code> sont tous traités en tant que <code>_blank</code> pour déterminer si le lien doit s'ouvrir ou non dans un nouvel onglet/fenêtre.</p>
+
+<div class="notecard note">
+  <h4>Note</h4>
```
Comment (SphinxKnight) [inline files/fr/web/html/link_types/noopener/index.html:14]: Preparing the field for Markdown conversion: could I ask to use a `<p><strong>Note :</strong> Blabla...</p>` for notes?

File: files/fr/web/html/link_types/noopener/index.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  <p>L'utilisation de <code>target="_blank"</code> sur des éléments <code>&lt;a&gt;</code> fournit désormais implicitement le même comportement pour l'attribut <code>rel</code> que si l'on avait utilisé <code>rel="noopener"</code>, c'est à dire l'absence de mise en place de <code>window.opener</code>. Voir la <a href="/fr/docs/Web/HTML/Element/a#browser_compatibility">compatibilité navigateur</a> pour plus d'information sur l'implémentation actuelle dans les navigateurs.</p>
```
Comment (SphinxKnight) [inline files/fr/web/html/link_types/noopener/index.html:15]: minor typo "pour plus d'information*s*"

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
A bunch of different things between translation itself, trivial typos and also "mdn arcanes" :x

**Comment by SphinxKnight** [comment]
> 
> 
> A bunch of different things between translation itself, trivial typos and also "mdn arcanes" :x

that being written: still a huge thank you :)

File: files/fr/web/html/link_types/noopener/index.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+title: 'Types de liens : noopener'
+slug: Web/HTML/Link_types/noopener
+browser-compat: html.elements.a.rel.noopener
+translation_of: 'Web/HTML/Link_types/noopener'
+---
+<p>Le mot-clé <strong><code>noopener</code></strong> de l'attribut <code><a href="/fr/docs/Web/HTML/Attributes/rel">rel</a></code> des éléments {{HTMLElement("a")}}, {{HTMLElement("area")}} et {{HTMLElement("form")}} indique au navigateur d'ouvrir la ressource cible sans donner d'accès au document précédent au nouveau contexte de navigation qui sera créé, en n'indiquant pas la propriété {{DOMxRef("Window.opener")}} sur la nouvelle fenêtre (il retourne <code>null</code>).</p>
```
[the remark this answers is not in the document]
Comment (audrasjb) [reply files/fr/web/html/link_types/noopener/index.html]: Fixed in https://github.com/mdn/translated-content/pull/1798/commits/0f0ad927b7f2548e2d12ccaabc29dff40c652eac with a few changes :)

File: files/fr/web/html/link_types/noopener/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/html/link_types/noopener/index.html]: Fixed in https://github.com/mdn/translated-content/pull/1798/commits/0f0ad927b7f2548e2d12ccaabc29dff40c652eac

File: files/fr/web/html/link_types/noopener/index.html (hunk 2 above)
Comment (audrasjb) [reply files/fr/web/html/link_types/noopener/index.html]: Ok ! Does the implementation proposed in https://github.com/mdn/translated-content/pull/1798/commits/0f0ad927b7f2548e2d12ccaabc29dff40c652eac work for you?

File: files/fr/web/html/link_types/noopener/index.html (hunk 3 above)
Comment (audrasjb) [reply files/fr/web/html/link_types/noopener/index.html]: 👍 fixed in https://github.com/mdn/translated-content/pull/1798/commits/0f0ad927b7f2548e2d12ccaabc29dff40c652eac

**Review by SphinxKnight — APPROVED** [review]

**Comment by SphinxKnight** [comment]
let's get this merged.

**Pull request merged**